### ISSUE TITLE
Xicam URL is a dead-end

### ISSUE THREAD

**Issue by prjemian** [nsls-ii-forge/xicam-feedstock#11]
Came here looking for Xi-CAM github source link.  Various place here use a repo off this organization (https://github.com/synchrotrons/) which gives a 404 not found error.

Where's the xicam repo?

**Comment by prjemian** [comment]
https://github.com/Xi-CAM

The URL mentioned above should be changed throughout this repo.

**Comment by mrakitin** [comment]
Good catch, thanks @prjemian! For some reason, GitHub does not redirect to the renamed orgs.

**Comment by mrakitin** [comment]
That should be addressed by #12 (may take some time to restructure the dependencies and create the missing feedstocks).
